governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Stablecoin Action Plan - November 2, 2020
+summary: Signal your support or opposition for executing the Stablecoin Action Plan.
+discussion_link: https://forum.makerdao.com/t/signal-request-stablecoin-action-plan/4808
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Stablecoin Action Plan - November 2, 2020
+
+The MakerDAO Governance Facilitator has placed a governance poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance community which gives Maker Governance the chance to signal their support or opposition for the implementation of the Stablecoin Action Plan.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-mcd-faqs/faqs#governance)) will be active for 3 days beginning on Monday, November 2 at 16:00 UTC.
+
+## Review
+
+Please refer to the [Stablecoin Action Plan thread](https://forum.makerdao.com/t/signal-request-stablecoin-action-plan/4808) for an overview of the proposed plan.
+
+`Option 3 - Migrate Users to a new Vault Type` won the signal request. If this poll passes, the Rates Working Group will work with the Smart Contracts Team to add an executive spell which will set both the Stability Fee and Debt Ceilings to 0 on the following vault types:
+
+ * USDC-A
+ * TUSD-A
+ * PAXUSD-A
+
+ This executive will then add the following vault types with the same settings, original Stability Fees and appropriate Debt Ceilings:
```

Comment:
I'm not sure if we want to commit to additional USDC vaults given it has the largest exposure. Maybe say that the exact new vaults added + parameters will be decided by the risk team?

Reply:
Not sure I understand. Are you saying you don't think we should commit to these particular vault types, or the "same settings"?